### ISSUE TITLE
Juicemind IDE does not run

### ISSUE THREAD

**Issue by KingLeonidas** [JuiceMind/JuiceMind#546]
When I or my students click on the run button, it stalls and stays on the syncing view and never actually runs the program. We sometimes have to refresh the page in order for it to run. but if we can any code and try to run it, it stalls again and again we need to refresh the page. My students mention that it is worst at their homes than at school.

----------------------------------------------------------------------------------------
LD: 1) Students during my PD 11 class were trying to run their programs but the output wouldn't show up.  I don't know if this is just a random glitch that happens occasionally.  

----------------------------------------------------------------------------------------
NW: Hello, two of my students are unable to run their programs under Unit 2 projects. The issue is in Period 8 and the programs that can't run are Slot Machine and Heron's Formula. One student cant run slot machine and it does say Java.
Another student is unable to even type inside the Guessing Game program without his browser being unresponsive.

----------------------------------------------------------------------------------------
MF: they hit run on their code and do not get any error messages or output
The program should output a .jpg file and it is not doing so
The assignment is Lab 2

---------------------------------------------------------------------------
DC: a student is getting an error on her page and is not able to see what she has created on her webpage. the files section is just spinning and will not load so I cant check to see if she is in java with swing

--------------------------------------------------------------------------
RN: Students are unable to run code. Connecting to server error

**Comment by juicemind-dev** [comment]
Hi @KingLeonidas this should be resolved now

**Comment by JL248** [comment]
I have this issue now.

**Comment by juicemind-dev** [comment]
Did you just test it @JL248 ?

**Comment by JL248** [comment]
Yes, I was trying to check my students' work 5 min ago and it was constantly spinning. In addition, yesterday while teaching online, my students and I often experienced spinning "Running.." message and had to wait for a long time to get the output for the programs.

**Comment by KingLeonidas** [comment]
It  is still happening today.

**Comment by karlr-dev** [comment]
@KingLeonidas @JL248 Hey guys this is fixed now.
